### ISSUE TITLE
kalhan:run not working with Kahlan 4.0.5

### ISSUE THREAD

**Issue by teol** [elephantly/kahlan-bundle#12]
Hi,

I just installed kahlan/kahlan (4.0.5) and elephantly/kahlan-bundle (v0.9.8) via composer on a Symfony 3.3 project.
The bundle is registered in AppKernel.php so I can execute the "kahlan:run" command but it's apparently not working as expected.

It throws an exception:
`[Symfony\Component\Debug\Exception\ClassNotFoundException]                                                                                      
  Attempted to load class "Filter" from namespace "Kahlan\Filter".  `

So I looked in the command file itself (KahlanRunCommand.php) to change this:
`use Kahlan\Filter\Filter;`
to this:
`use Kahlan\Filter\Filters as Filter;`

But even as a quick workaround it's not working:
`[Symfony\Component\Debug\Exception\UndefinedMethodException]                              
  Attempted to call an undefined method named "register" of class "Kahlan\Filter\Filters".`

**Comment by Tiriel** [comment]
Hey! Thanks for the report!

Could you provide a bit more context? Like, could it be possible to see an example of test file you were attempting to run?

Thanks again, I'll check this ASAP (probably this week-end)

**Comment by teol** [comment]
Well I was just trying to discover Kahlan on a Symfony project. The project itself is not empty but I had no Kahlan test at this time (my "spec" folder was empty).
Maybe that was the problem ?

I think the Exception really comes from the command itself, to me it looked like something wasn't up-to-date.

Also, I haven't made any config file for Kahlan, but I could execute the Kahlan binary without error.

I hope this helps

**Comment by samsonasik** [comment]
@teol duplicate with #11 . kahlan 4 have different behaviour for filters, you can setup the filters for it like I blogged about it at https://samsonasik.wordpress.com/2018/01/03/functional-test-symfony-4-with-kahlan-4/

**Comment by teol** [comment]
Yeah that's actually your article that made me want to try Kahlan 😃 (and thanks for writing it by the way).
I'm going to try again with your config file.
